### ISSUE TITLE
Issues with vue-cli 3

### ISSUE THREAD

**Issue by yanfali** [officert/vue-slideout-panel#24]
Hello, I was unable to get your instructions working with the very latest version of vue-cli. For whatever reason with webpack the plugin would not register itself correctly. I ended up hacking around the instructions

```javascript
import * as Slideout from 'vue2-slideout-panel';
Vue.component('slideout-panel', Slideout.VueSlideoutPanel);
```

Not really sure what I'm doing wrong.

I also had to invoke it in a similar manner

```javascript
Slideout.vueSlideoutPanelService.show
```

As it wasn't able to install itself correctly. Awesome plugin though once I got it working

**Comment by officert** [comment]
@yanfali can you try v.1.0.8?

Here's the change I made: https://github.com/officert/vue-slideout-panel/commit/48ea60d7031613f394159eff69f6bd194821a9f8#diff-1fdf421c05c1140f6d71444ea2b27638R19

I wasn't installing up the component properly.

https://github.com/officert/vue-slideout-panel/releases/tag/1.0.8

**Comment by yanfali** [comment]
I'll give it a shot. Thanks

**Comment by yanfali** [comment]
looks like the code load cache is still refreshing, so it's only showing 1.07 right now in yarn. I'll give it another try in the morning.

**Comment by officert** [comment]
@yanfali sorry about that, looks it didn't publish. Try now, Yarn is showing the right version now: https://yarnpkg.com/en/package/vue2-slideout-panel

**Comment by yanfali** [comment]
@officert can you republish to npm please? It's still showing as 1.0.7 only. Thank you.

**Comment by yanfali** [comment]
got it, thanks

**Comment by yanfali** [comment]
@officert having trouble with the examples, it's crashing when I invoke

const panelInstance = this.$showPanel.show({
  component : 'your-component-name',
  props: {
    //any data you want passed to your component
  }
});

this.$showPanel is not available from my component.

```javascript
<script>
require('milligram');
import Vue from 'vue';
import { isUndefined } from 'lodash';
import { VueSlideoutPanel } from 'vue2-slideout-panel';

Vue.component('slideout-panel', VueSlideoutPanel);

export default {
  data() {
    return {
      watcher: undefined
    };
  },
  components: { VueSlideoutPanel },
  mounted() {
    this.watcher = this.$store.watch(
      (state, getters) => getters.panelVisible,
      this.showPanel
    );
  },
  beforeDestroy() {
    if (!isUndefined(this.watcher)) {
      this.watcher();
    }
  },
  methods: {
    showPanel(panelVisible) {
      console.log(panelVisible);
      if (panelVisible) {
        this.$showPanel.show({
          component: 'story-detail-panel',
          openOn: 'right',
          props: {},
          width: '600px'
        });
      } else {
        this.$emit('closePanel', {});
      }
    }
  }
};
</script>
```

**Comment by ChadTaljaardt** [comment]
@yanfali Did you find a solution this your latest issue? I am getting it now as well.

**Comment by officert** [comment]
@yanfali sorry for the slow response. I figured out the issue. I wasn't properly install the component with Vue. I've fixed that issues and release V2.0.0. 

https://github.com/officert/vue-slideout-panel/releases/tag/2.0.0

Take a look at the release notes as there are breaking changes. You'll just need to make sure your install the component properly, then you will be able to use `this.$showPanel()` from inside all your components.

Let me know if you have any issues
